Tools/WebKitTestRunner/InjectedBundle/mac/AccessibilityUIElementMac.mm

```diff
@@ -542,6 +546,34 @@ void setAttributeValue(id element, NSString* attribute, id value, bool synchrono
     return AccessibilityUIElement::create(element.get());
 }
 
+RefPtr<AccessibilityUIElement>  AccessibilityUIElement::elementAtPointWithRemoteElementForTesting(int x, int y)
+{
+    RetainPtr<id> element;
+    s_controller->executeOnAXThreadAndWait([&x, &y, &element, this] {
+        element = [m_element _accessibilityHitTest:NSMakePoint(x, y) returnPlatformElements:NO];
+    });
+
+    if (!element)
+        return nullptr;
+
+    return AccessibilityUIElement::create(element.get());
```

Comment:
Could be a bit shorter as:

`return element ? AccessibilityUIElement::create(element.get()) : nullptr;`

Reply:
Unfortunately the compiler doesn't agree with this one, so going to have to leave it as-is for now.